firstvoices/backend/resources/utils/import_export_widgets.py

```diff
@@ -117,18 +120,13 @@ def clean(self, value, row=None, **kwargs):
         ]
 
 
-class CleanForeignKeyWidget(ForeignKeyWidget):
-    def __init__(self, model, field, title_case=False, *args, **kwargs):
-        self.title_case = title_case
-        super().__init__(model=model, field=field, *args, **kwargs)
+class PartOfSpeechWidget(ForeignKeyWidget):
```

Comment:
Can be later renamed again as `CleanForeignKeyWidget` and made reusable if required, but was currently only used here so I put the PartOfSpeech in the name.